scripts/generators/readmes.js

```diff
@@ -15,13 +15,35 @@ const packageDir = join(cwd, "packages");
 const packages = readdirSync(packageDir);
 const getWebsiteLink = n => `https://new.babeljs.io/docs/en/next/${n}.html`;
 const getPackageJson = pkg => require(join(packageDir, pkg, "package.json"));
-
-const generateReadme = ({ link, name, description }) =>
+const getIssueLabelLink = l =>
+  `https://github.com/babel/babel/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22${encodeURIComponent(
+    l
+  )}%22`;
+
+const labels = {
+  "babel-preset-typescript": getIssueLabelLink("area: typescript"),
+  "babel-preset-flow": getIssueLabelLink("area: flow"),
+  "babel-preset-react": getIssueLabelLink("area: react"),
+  "babel-parser": getIssueLabelLink("pkg: babylon"),
+  "babel-cli": getIssueLabelLink("pkg: cli"),
+  "babel-core": getIssueLabelLink("pkg: core"),
+  "babel-polyfill": getIssueLabelLink("pkg: polyfill"),
+  "babel-preset-env": getIssueLabelLink("pkg: preset-env"),
+  "babel-regiser": getIssueLabelLink("pkg: regiser"),
+  "babel-traverse": getIssueLabelLink("pkg: traverse"),
+  "babel-types": getIssueLabelLink("pkg: types"),
+};
```

Comment:
Let's make a beginner-friendly issue to add more here

Reply:
> Also would be interesting to link to the spec: xx ones?

Yup!